ass3.html

```diff
@@ -0,0 +1,57 @@
+<html>
+
+<head>
+    <title> Assignment 3</title>
+</head>
+
+<body>
+    <div>
+        <button type="button" onclick="operatetabs(1)">Open Tabs</button>
+        <button type="button" onclick="operatetabs(2)">Close Tabs</button>
+
+
+    </div>
+
+
+    <script>
+        let arr = [];
+        const operatetabs = (ch) => {
+
+            switch (ch) {
+                case 1:
+                    for (let i = 0; i < 5; i++) {
+                        let ref = window.open("https://www.google.com");
+                        arr.push(ref);
+                        
+                    }
+
+                    setTimeout(() => {
+                        for (let i = 4; i >= 0; i--) {
+
+                            arr[i].close();
+                            arr.pop();
+
+                        }
+                    }, 10000);
+                    
+                    break;
+
+                case 2:
+
+                    for (let i = 0; i < 5; i++) {
```

Comment:
Open windows should be close one by one after every 5 seconds and the closing of windows should be stopped on the click stop button.